app/controllers/concerns/shopify_app/domain_protection.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ShopifyApp
+  module DomainProtection
```

Comment:
Just want to call attention to the name for other reviewers. We wanted to name this something that indicates its responsibility and reflects that it would be used instead of `LoginProtection`. Does this name do that?

Reply:
Maybe `RequireKnownShop` or something would be more clear what it does?